### PR TITLE
Fix cpptango 440 document group enable disable

### PR DESCRIPTION
**Pull request by mliszcz**
Fixes tango-controls/cppTango#440.

### CODE REVIEW

File: source/development/client-api/cpp-client-programmers-guide.rst [hunk 1]
```diff
@@ -1058,6 +1058,42 @@ alive.
...
+    gauges->disable("inst-c01/vac-gauge/pirani-01", not forwarded);
+
+    // will disable: inst-c01/vac-gauge/pirani-01
+    gauge_family->disable("inst-c01/vac-gauge/pirani-01");
+
+    // will enable: inst-c01/vac-gauge/penning-01
+    gauges->enable("inst-c01/*");
+
+    // entries for disabled elements will be included in the result set,
+    // however they will not have a value and will be marked as disabled
```
Comment (bourtemb) [inline source/development/client-api/cpp-client-programmers-guide.rst]: How are they marked as disabled? Exception? A flag?
Can we add an example here showing how the user can ignore the attributes which are disabled in the result set?

File: source/development/client-api/cpp-client-programmers-guide.rst [hunk 2]
```diff
@@ -1058,6 +1058,42 @@ alive.
...
+Devices belonging to a group can be temporarily excluded from all operations
+performed on the group using the *Group::disable* and *Group::enable* calls.
+
+Device name passed to the *disable* (*enable*) methods can contain
+wildcards (``*``).
+Note that only the first matching device will be disabled (enabled).
+
+The search algorithm is breadth-first search. All group elements are searched
+(in the insertion order) for a match before descending recursively to
+sub-groups. Recursive search can be disabled with a flag.
```
Comment (bourtemb) [inline source/development/client-api/cpp-client-programmers-guide.rst]: If I understood well, this flag is the fwd optional boolean parameter (with default value = true) which can be provided to enable() and disable() methods.
Can we add this information somehow?

File: source/development/client-api/cpp-client-programmers-guide.rst (hunk 1 above)
Comment (mliszcz) [reply source/development/client-api/cpp-client-programmers-guide.rst]: I've added `group_element_enabled()` check to the example. Is it ok now?

File: source/development/client-api/cpp-client-programmers-guide.rst (hunk 2 above)
Comment (mliszcz) [reply source/development/client-api/cpp-client-programmers-guide.rst]: I've added a reference to the "forwarding" section to this paragraph. Could you please check if it's ok now?

**Pull request merged**